src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -209,4 +242,32 @@ namespace ts.codefix {
             changeTracker.insertNodeAtClassStart(classDeclarationSourceFile, classDeclaration, methodDeclaration);
         }
     }
+
+    function createEnumMemberFromEnumDeclaration(checker: TypeChecker, token: Identifier, enumDeclaration: EnumDeclaration) {
+        /**
+         * create initializer only string enum.
+         * value of initializer is a string literal that equal to name of enum member.
+         * literal enum or empty enum will not create initializer.
+         */
+        const firstMember = firstOrUndefined(enumDeclaration.members);

```

Comment:
this should be `some`, and not `first`. a literal enum can have both members, e.g.:

```ts
enum E { 
    a,
    b = 1,
    c = "ss"
}
```